benchmark/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 # CMake info 
 #-------------------------------------------------------------------------------------------
 cmake_minimum_required(VERSION 3.1)
-project(EASTLBenchmarks)
+project(EASTLBenchmarks CXX)
```

Comment:
Only skips the detection for a C compiler and just checks for a working C++ compiler.
Probably just my personal taste than any real benefit.